src/table/src/TableRow.js

```diff
@@ -92,7 +92,7 @@ class TableRow extends PureComponent {
     if (this.props.isSelectable) {
       const { key } = e
       if (key === 'Enter' || key === ' ') {
-        this.props.onSelect()
+        this.props.onSelect(e)
```

Comment:
To bubble the event you may want to use [`event.persist()`](https://reactjs.org/docs/events.html) first:

>If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.

Reply:
That's a good point, but is this async code? I thought these callbacks were synchronous, CMIIW. I noticed that we're already passing the event on to other handlers in this function without `event.persist()`.